### PR TITLE
jan gems

### PR DESCRIPTION
**Pull request by elleobrien**
here's the draft, everything is ready but hte cover

### CODE REVIEW

File: content/blog/2021-01-20-january-21-dvc-heartbeat.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+tags:
+  - Heartbeat
+  - CML
+  - DVC
+  - DAGsHub
+  - R
+  - MLOps
+---
+
+## News
```
Comment (shcheklein) [inline content/blog/2021-01-20-january-21-dvc-heartbeat.md:30]: I guess no need for this H2 if it's the only one?

File: content/blog/2021-01-20-january-21-dvc-heartbeat.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
+---
+title: January '21 Heartbeat
+date: 2021-01-20
+description: |
+  Monthly updates are here- read all about
```
Comment (shcheklein) [inline content/blog/2021-01-20-january-21-dvc-heartbeat.md:5]: 💅 formatting looks a bit off

File: content/blog/2021-01-20-january-21-dvc-heartbeat.md [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+
+Watch and follow along! If you make something based on this approach, or if you
+think there's a better way, please tell us- we're eager to see what the R
+community thinks.
+
+https://youtu.be/NwUijrm2U2w
+
+### Workshops and talks
+
+Coming up Friday, Jan 21, I (Elle) will be speaking with
```
Comment (shcheklein) [inline content/blog/2021-01-20-january-21-dvc-heartbeat.md:75]: Friday Jan 22? not relevant anymore? :(

Comment (elleobrien) [reply content/blog/2021-01-20-january-21-dvc-heartbeat.md:75]: ah yeah. i'll change the tense to make it "check out this thing we just recorded!"

File: content/blog/2021-01-20-january-21-dvc-heartbeat.md [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+[Alexey Grigorev](https://twitter.com/Al_Grigor), author of a
+[Data Science Bookcamp](https://mlbookcamp.com/), on his podcast about being a
+developer advocate in the machine learning space! If you're curious about what
+the role entails, or what to look for when hiring a developer advocate for your
+machine learning project, please come by. The event will be recorded live and
+then shared for your listening pleasure 🎧
+
+https://twitter.com/Al_Grigor/status/1351955523890012167
+
+## From the community
```
Comment (shcheklein) [inline content/blog/2021-01-20-january-21-dvc-heartbeat.md:85]: Oh, I see the second H2 here, sorry about the previous comment - you can disregard.

**Review by shcheklein — APPROVED** [review]
Great stuff! Some minor comments, if header/meta are up to date and ready feel free to merge!

**Pull request merged**